### DISCUSSION TITLE
Use a fixed y position for the dataPointLabelComponent

### DISCUSSION THREAD

**Discussion by dannybit** [Abhinandan-Kushwaha/react-native-gifted-charts#836] [Q&A]
Hi, thank you again for the great library. I am using dataPointLabelComponent to render a modal when the user focuses on a datapoint. 

<img width="415" alt="Screenshot 2024-09-15 at 3 58 30 PM" src="https://github.com/user-attachments/assets/2a5edbb1-9d60-4203-a966-8c0f5a4792d0">

I want to always show the modal at the top of the chart, regardless of the position of the actual datapoint.
I can achieve this using `dataPointLabelShiftY`, but the value would need to depend on the initial position of the dataPointLabel, which varies per dataPoint.

<img width="436" alt="Screenshot 2024-09-15 at 4 01 08 PM" src="https://github.com/user-attachments/assets/7528785d-c0ed-4f7c-8c29-99b52f8df0d8">

Is there a way to use a fixed Y position for the dataPointLabel across all data points?

**Comment by Abhinandan-Kushwaha** [comment]
I think this is not supported using `dataPointLabelComponent` but can be achieved using `pointerConfig` instead of using the focus approach.

**Comment by dannybit** [comment]
Thanks, I'll give the `pointerConfig` a try. In my graph I have a lot of `undefined` (interpolated) datapoints. With the pointerConfig is there a way to not render the strip and labelComponent for those datapoints?

**Comment by dannybit** [comment]
I can confirm that this can be achieved with `pointerConfig` but unfortunately I can't use it because the pointer also points to interpolated and extrapolated values in the chart.
